docs/pages/monitoring-pelican-services/prometheus.mdx

```diff
@@ -187,17 +190,102 @@ All of the Pelican servers have the following metrics:
 
   #### Label: `server_lat`
 
-  The storage server latitute.
+  The storage server latitude.
 
   #### Label: `server_long`
 
-  The storage server longitute.
+  The storage server longitude.
 
 
 ### `# of Active Origins and Caches`
 
 With the `up` metric, it is possible to count number of active origin and cache servers in the federation by a simple Prometheus query: `count(up{server_type=<"Origin">})` for counting origin servers, or `count(up{server_type=<"Cache">})` for counting cache servers.
 
+### `pelican_director_advertisements_received_total`
+
+  The accumulated number of origin/cache advertisements to the director. This metric shows if an origin/cache server successfully joins the federation or not. For origin servers, it also shows if each federation namespace prefix it exports passed director verification.
+
+  #### Label: `server_name`
+
+  The name of the storage server. By default it's the hostname.
+
+  #### Label: `server_type`
+
+  | Label Values | Description     |
+  |--------------|-----------------|
+  | `Origin`     | Origin server   |
+  | `Cache`      | Cache server    |
+
+  #### Label: `server_web_url`
+
+  The storage server web url.
+
+  ### Label: `namespace_prefix`
+
+  The federation namespace prefix the storage server exported.
+
+  ### Label: `status_code`
+
+  The status code of the director's response. The most useful value is `403`, which means the server advertisement didn't pass director's verification.
+
+  | Label Values | Description     |
+  |--------------|-----------------|
+  | `200`     | Advertisement succeeded   |
+  | `403`      | Advertisement verification failed    |
+  | `500`      | Director has errors when verify and save the advertisement    |
+
+### `pelican_director_stat_total`
+
+  The accumulated number of `stat` query the director made to origin/cache servers to check for object availability. Only available when `Director.EnableStat` is set to true. This metric is a good indicator of object availability and origin/cache service quality.
+
+  #### Label: `server_name`
```

Comment:
Is this repeated from line 208?

Reply:
Nope, they differ in different labels, and I decided to explicitly list all out instead of referring back to existing labels for better readability